src/index.html

```diff
@@ -2,17 +2,92 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+
     <title>Product cards</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__image">
+        <img
+          class="card__image-photo"
+          src="images/imac.jpeg"
+          alt="product photo"
+        />
+      </div>
+
+      <div class="card__model">
+        <h2 class="card__model-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+      </div>
+
+      <div class="card__product-code">
+        <h3 class="card__product-code-style">Product code: 195434</h3>
+      </div>
```

Comment:
The product code can be marked up as a paragraph (`<p>`) instead of a heading (`<h3>`), as it is not a title of a section. Semantic HTML elements should be used appropriately to improve accessibility and SEO.